<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Sairam" ignored="1" name="Default test(failed)" tests="2" failures="1" timestamp="2020-04-30T12:18:10 IST" time="0.151" errors="0">
  <testcase name="test2" time="0.02" classname="demo.TestNGRetryFailedDemo"/>
  <testcase name="test3" time="0.009" classname="demo.TestNGRetryFailedDemo">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at demo.TestNGRetryFailedDemo.test3(TestNGRetryFailedDemo.java:24)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase name="test1" time="0.0" classname="demo.TestNGRetryFailedDemo">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Default test(failed) -->
